packages/floating-vue/src/components/Popper.ts

```diff
@@ -643,6 +640,9 @@ export default () => defineComponent({
 
       const data = await computePosition(this.$_referenceNode, this.$_popperNode, options)
 
+      // Emit resize-Event evevry time position is re-computed to replace onResize.
+      this.$emit('resize')
```

Comment:
This should probably be in a specific callback on àutoUpdate`?

Reply:
Sure, that sounds good. I'll change it after lunch.